packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -169,6 +175,23 @@ export const Webchat = forwardRef((props, ref) => {
   const { theme } = webchatState
   const { initialSession, initialDevSettings, onStateChange } = props
   const [botonicState, saveState, deleteState] = useLocalStorage('botonicState')
+  const saveWebchatState = webchatState => {
+    // Serialization of regexs: https://stackoverflow.com/questions/12075927/serialization-of-regexp
```

Comment:
imho this comment should be in serializeRegexs.
Maybe it's better to have a "function serializeWithRegexs(object): string" which does 
{
const serializeRegex= (_, value) {...}
return JSON.stringify(object,  serializeRegex)
}